### ISSUE TITLE
bug: Upgraded to asdf 0.9.0  with Homebrew - command-help.bash file not found

### ISSUE THREAD

**Issue by josephearl** [asdf-vm/asdf#1161]
### Describe the Bug

libexec/bin/asdf: line 82: /usr/local/opt/asdf/lib/commands/command-help.bash: No such file or directory

In my .zshrc I have set

export ASDF_DATA_DIR="$HOME/.asdf"
export ASDF_DIR="$(brew --prefix asdf)"
source "$ASDF_DIR/libexec/asdf.sh"

### Steps to Reproduce

Upgraded from older version of asdf with `home-brew upgrade asdf`

### Expected Behaviour

It works and the command directory and files exist

### Actual Behaviour

It does not work and the commands directory does not exist

lib directory only contains 2 files:

```
ls /usr/local/opt/asdf/lib
asdf.fish asdf.sh
```

Unknown command: `asdf which java`                                                                       
/usr/local/Cellar/asdf/0.9.0/libexec/bin/asdf: line 82: /usr/local/opt/asdf/lib/commands/command-help.bash: No such file or directory

### Environment

```shell
asdf does not work
```

**Comment by josephearl** [comment]
I have tried uninstalling and re-installing asdf, still there is no commands directory in lib

**Comment by josephearl** [comment]
After manually copying that folder from source I get another error:

```
cat: /usr/local/opt/asdf/help.txt: No such file or directory
```

**Comment by josephearl** [comment]
After manually copying the help.txt it works, but none of this should be necessary - all these files should be installed with Homebrew

**Comment by josephearl** [comment]
In the asdf--0.9.0.all.bottle.tar.gz it seems these are in libexec/lib/commands and lib/commands

**Comment by josephearl** [comment]
So removing `export ASDF_DIR="$(brew --prefix asdf)"` fixed most of the issues exception from the warnings from plugin commands not existing when I run `asdf help`

**Comment by RobLoach** [comment]
For anyone else Google-ing around for this issue, this find print error was fixed a couple days after the release of 0.9.0 over at https://github.com/asdf-vm/asdf/pull/1102 .
